Styles/styles.css

```diff
@@ -0,0 +1,410 @@
+}
+
+.triangle-background-3 {
+    width: 100%;
+    border-bottom: 50px solid white;
+    border-right: 1500px solid transparent;
+}
+
+.triangle-background-4 {
+    width: 100%;
+    border-bottom: 50px solid transparent;
+    border-right: 1500px solid white;
+}
+    /* ------------------------------- 2. Typography ----------------------------------- */
+
+    h1 {
+        font-family: "Merriweather", serif;
+        font-weight: 700;
+        margin: 30px;
+        font-size: 44px;
+    }
+
+    h2 {
+        font-family: "Merriweather", serif;
+        font-weight: 700;
+        font-size: 32px;
+        margin: 30px 0;
+    }
+
+    h3 {
+        font-family: "Merriweather", serif;
+        font-size: 27px;
+        font-weight: 300;
+        color: white;
+    }
+
+    body {
+        font-family: "Roboto", serif;
+        font-weight: 200;
+        font-style: normal;
+        margin: 0 auto;
+    }
+
+    main {
+        max-width: 1600px;
+        color: black;
+    }
+
+    footer {
+        background-color: black;
+        color: white;
+        height: 70px;
+        max-width: 1500px;
+        border-bottom-left-radius: 5px;
+        border-bottom-right-radius: 5px;
+        margin: 0 auto 40px auto;
+    }
+    /* ------------------------------- 3. Layout --------------------------------------- */
+
+    /* --------------- 3.1 Reusables ----------------------- */
+
+    /* --------------- 3.2 Other --------------------------- */
+
+    .form-section {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: center;
+        height: 860px;
+        background: linear-gradient(158deg, rgba(72,69,228,1) 61%, rgba(87,34,205,1) 95%);
+        max-width: 1500px;
+        color: white;
+        margin: 0 auto;
+    }
+
+    .h2-form {
+        display: flex;
+        flex-direction: column;
+        justify-items: flex-start;
+        align-items: center;
+        margin: 0 auto 30px auto;
+    }
+
+    .h2-form p {
+        font-size: 20px;
+        font-weight: 300;
+    }
+
+    .h2-form h2 {
+        font-family: "Merriweather", serif;
+        font-weight: 700;
+        font-size: 44px;
+        padding-top: 80px;
+    }
+
+    .input-area {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        align-items: flex-start;
+        width: 850px;
+    }
+
+    .input-area input {
+        margin-bottom: 20px;
+        width: 850px;
+        height: 65px;
+        justify-items: center;
+        padding-left: 20px;
+        border-radius: 3px;
+        border-style: none;
+        font-family: "Roboto", serif;
+        color: #7A7A7A;
+        font-weight: 400;
+        font-size: 16px;
+        /*text-shadow: 0 0 1px ;*/
+    }
+
+    .input-area textarea {
+        margin: 0 auto;
+        width: 850px;
+        flex-basis: 220px;
+        justify-items: flex-start;
+        padding-left: 15px;
```

Comment:
Padding kan je 4 waardes geven, namelijk: top, right, bottom, left.

Je zou het dus zo kunnen schrijven: padding: 15px 0 0 15px;
Dat is korter dan de 2 regels code die je nu hebt staan:
padding-left: 15px;
padding-top: 15px;